modules/osdk-upgrading-v1-4-0.adoc

```diff
@@ -0,0 +1,167 @@
+
+. For Ansible- and Helm-based Operator projects, add liveness and readiness probes.
++
+New projects built with the Operator SDK will have the probes configured by default. The endpoints `/healthz` and `/readyz` are available now in the provided image base. You can update your existing projects to use them by updating the `Dockerfile` to use the latest base image, then add the following to the `manager` container in the `config/manager/manager.yaml` file:
++
+.Example for Ansible-based Operator projects
+[source,yaml]
+----
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 6789
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 6789
+    initialDelaySeconds: 5
+    periodSeconds: 10
+----
++
+.Example for Helm-based Operator projects
+[source,yaml]
+----
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 8081
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 8081
+    initialDelaySeconds: 5
+    periodSeconds: 10
+----
+
+. For Go-based Operator projects, update the `PROJECT` file `plugins` object to use `manifests` and `scorecard` objects.
++
+The `manifests` and `scorecard` plug-ins that create Operator Lifecycle Manager (OLM) and scorecard manifests, respectively, now have plug-in objects that direct `create` subcommands to create related files. While the old Go-based plug-in configuration object is still supported, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
++
+.Old configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+----
++
+.New configuration
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+. For Ansible- and Helm-based Operator projects, add `manifests` and `scorecard` objects to the `Makefile`.
++
+The `manifests` and `scorecard` plug-ins that create Operator Lifecycle Manager (OLM) and scorecard manifests, respectively, now have plug-in objects that direct `create` subcommands to create related files. While not necessary to add, these new objects will be useful in the future as configuration options are added to their respective plug-ins:
++
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+. Move the positional directory argument `.` in the `make` target for `docker-build`.
++
+The directory argument `.` in the `docker-build` target was moved to the last positional argument to align with `podman` CLI expectations, which makes substitution cleaner:
++
+.Old target
+[source,terminal]
+----
+docker-build:
+  docker build . -t ${IMG}
+----
++
+.New target
+[source,terminal]
+----
+docker-build:
+  docker build -t ${IMG} .
```

Comment:
would it be helpful to mention the `sed` command with which the users can make this change? Like:
```
sed -i 's/docker build . -t ${IMG}/docker build -t ${IMG} ./' $(git grep -l 'docker.*build \. ')
```
Reference PR: https://github.com/operator-framework/operator-sdk/pull/4466